packages/kit/src/views/ReferFriends/pages/InviteReward/components/RewardCard/InfoIcon.tsx

```
@@ -0,0 +1,19 @@
+  onPress?: () => void;
+  size?: string;
+  tooltip?: string;
+}
+
+export function InfoIcon({ onPress, size = '$5', tooltip }: IInfoIconProps) {
+  const icon = <Icon name="InfoCircleOutline" size={size} onPress={onPress} />;
+
+  if (tooltip) {
+    return (
+      <Tooltip renderTrigger={icon} renderContent={tooltip} />
+    );
```

ESLint

[error] Replace `(⏎······<Tooltip·renderTrigger={icon}·renderContent={tooltip}·/>⏎····)` with `<Tooltip·renderTrigger={icon}·renderContent={tooltip}·/>`

(prettier/prettier)